great_expectations/data_context/store/datasource_store.py

```diff
@@ -354,3 +363,28 @@ def add_batch_config(self, batch_config: BatchConfig) -> BatchConfig:
 
         updated_batch_config._data_asset = updated_asset
         return updated_batch_config
+
+    def delete_batch_config(self, batch_config: BatchConfig) -> None:
```

Comment:
what's our convention around what these methods should return? i've been returning the deleted object in `ExpectationsStore` - don't have a strong opinion either way right now, but let's coordinate for consistency.

Reply:
I don't have strong opinions either, but my thinking was that add returns an object representing what is now persisted, and for parity, delete_* should return what's persisted: nothing.